VideoWeb/VideoWeb/ClientApp/src/app/waiting-space/waiting-room-shared/waiting-room-base.component.ts

```diff
@@ -820,7 +814,7 @@ export abstract class WaitingRoomBaseDirective implements AfterContentChecked {
         let pexipNode = this.hearing.getConference().pexip_node_uri;
         let conferenceAlias = this.hearing.getConference().participant_uri;
         let displayName = this.participant.tiled_display_name;
-        if (this.needsInterpreterRoom() && !this.vodafoneEnabled) {
+        if (this.needsInterpreterRoom() && this.conference.supplier === Supplier.Kinly) {
```

Comment:
this section can be removed now because:

1. We won't be using participant VMRs again
2. Vodafone implementation doesn't support it so it might be safer to remove this obsolete code now